src/opat.ts

```diff
@@ -10,14 +10,19 @@
  *
  * Example commands:
  *   npx ts-node src/opat.ts validate -f tests/examples/valid.yaml # Output: Valid!
- *   npx ts-node src/opat.ts validate -f tests/examples/invalid.yaml # Output: Invalid: data must have required property 'title'
+ *   npx ts-node src/opat.ts validate -f tests/examples/invalid-basic.yaml # Output: Invalid: ...
+ *   npx ts-node src/opat.ts validate -f tests/examples/valid.yaml -cf catalog/wcag2-catalog.yaml # Output: Valid!
+ *   npx ts-node src/opat.ts validate -f tests/examples/invalid-criteria.yaml -cf catalog/wcag2-catalog.yaml # Output: Invalid: ...
+ *   npx ts-node src/opat.ts validate -f tests/examples/invalid-components.yaml --cf catalog/wcag2-catalog.yaml # Output: Invalid: ...
+ *   npx ts-node src/opat.ts validate -f tests/examples/invalid-components-criteria.yaml --cf catalog/wcag2-catalog.yaml # Output: Invalid: ...
  */
 
 import { validateOPAT } from "./validateOPAT";
 import yargs from "yargs";
 import fs from "fs";
 import yaml from "js-yaml";
 import { ValidatorResult } from "./ValidatorResult";
+import { validateOPATCatalogValues } from "./validateOPATCatalogValues";
```

Comment:
I like that we are breaking these out - ultimately I think it should be possible to use these as an NPM library, as well as a CLI.